engine/job.go

```diff
@@ -113,23 +104,11 @@ func (job *Job) CallString() string {
 	return fmt.Sprintf("%s(%s)", job.Name, strings.Join(job.Args, ", "))
 }
 
-func (job *Job) StatusString() string {
-	// If the job hasn't completed, status string is empty
-	if job.end.IsZero() {
-		return ""
-	}
-	var okerr string
-	if job.status == StatusOK {
-		okerr = "OK"
-	} else {
-		okerr = "ERR"
-	}
-	return fmt.Sprintf(" = %s (%d)", okerr, job.status)
-}
-
 // String returns a human-readable description of `job`
 func (job *Job) String() string {
-	return fmt.Sprintf("%s.%s%s", job.Eng, job.CallString(), job.StatusString())
+	// why do we need to print exit err/status when maybe job isn't done?
```

Comment:
feel free to ask questions on IRC or github comments, not code comments :)
Where is this function called from? It's hard to grep for it. Would you mind checking if this function is needed at all? (just add a panic in this function to see that stacktrace)